Libraries/TestHelpers.psm1

```diff
@@ -58,12 +58,15 @@ function Upload-RemoteFile($uploadTo, $port, $file, $username, $password, $usePr
 	if (!$maxRetry) {
 		$maxRetry = 10
 	}
-
+	if ($sshPublicKey -and $sshPrivateKey) {
```

Comment:
do they come from global variable? can we make it more clear by adding $global: ?